lisa/operating_system.py

```diff
@@ -547,16 +547,18 @@ def _get_version_info_from_named_regex_match(
             patch_match = "0"
         major_match = named_matches.group("major")
         minor_match = named_matches.group("minor")
-        build_match = named_matches.group("build")
         major, minor, patch = map(
             int,
             [major_match, minor_match, patch_match],
         )
         build_match = named_matches.group("build")
-        self._node.log.debug(
+        log_message = (
             f"Found {package_name} version "
-            f"{major_match}.{minor_match}.{patch_match}-{build_match}"
+            f"{major_match}.{minor_match}.{patch_match}"
         )
+        if build_match:
```

Comment:
The log message can be clearer like:

f"Found {package_name} version "
            f"major: {major_match}, minor: {minor_match}, patch: {patch_match}, build: {build_match}"

Reply:
Nice, I agree! Thanks